src/pages/project/project_wizard/EmptyProjectForm.tsx

```diff
@@ -1,3 +1,56 @@
-export const EmptyProjectForm = () => {
-  return <p>Empty project</p>;
+import EastOutlinedIcon from '@mui/icons-material/EastOutlined';
+import InsertDriveFileOutlinedIcon from '@mui/icons-material/InsertDriveFileOutlined';
+import { Box, Button, TextField } from '@mui/material';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { SaveCristinProject } from '../../../types/project.types';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { CreateProjectAccordion } from './CreateProjectAccordion';
+
+interface EmptyProjectFormProps {
+  newProject: SaveCristinProject;
+  setNewProject: (val: SaveCristinProject) => void;
+  setShowProjectForm: (val: boolean) => void;
+}
+
+export const EmptyProjectForm = ({ newProject, setNewProject, setShowProjectForm }: EmptyProjectFormProps) => {
+  const { t } = useTranslation();
+  const [title, setTitle] = useState('');
+  const disabled = !title;
+
+  const createProject = () => {
+    setNewProject({ ...newProject, title: title });
+    setShowProjectForm(true);
+  };
+
+  return (
+    <CreateProjectAccordion
+      headline={t('project.form.start_with_empty_form_headline')}
+      description={t('project.form.start_with_empty_form')}
+      testId={dataTestId.newProjectPage.createEmptyProjectAccordion}
+      icon={<InsertDriveFileOutlinedIcon sx={{ height: '3rem', width: '3rem', mr: '0.75rem' }} />}>
+      <Box sx={{ display: 'flex', flexDirection: 'column' }}>
```

Comment:
Siden det ligger en flex her er det kanskje naturlig at den også styrer gaps i stedet for at det er i `mt` på knappen?
```suggestion
      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
```